src/Illuminate/Database/Migrations/Migrator.php

```diff
@@ -591,4 +615,33 @@ protected function note($message)
             $this->output->writeln($message);
         }
     }
+
+    /**
+     * Set the event dispatcher instance.
+     *
+     * @param  \Illuminate\Contracts\Events\Dispatcher  $dispatcher
+     * @return void
+     */
+    public function setEventDispatcher(Dispatcher $dispatcher)
+    {
+        static::$dispatcher = $dispatcher;
+
+        return $this;
+    }
+
+    /**
+     * Fire the given event for the model.
+     *
+     * @param  string  $event
+     * @param  array  $params
+     * @return mixed
```

Comment:
Make this method void.
```suggestion
     * @return void
```

Reply:
Done